cmds/ocm/commands/ocmcmds/common/inputs/types/npm/input_test.go

```diff
@@ -0,0 +1,31 @@
+package npm_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "ocm.software/ocm/cmds/ocm/commands/ocmcmds/common/inputs/testutils"
+	"ocm.software/ocm/cmds/ocm/commands/ocmcmds/common/inputs/types/npm"
+
+	"ocm.software/ocm/cmds/ocm/commands/ocmcmds/common/inputs"
+	"ocm.software/ocm/cmds/ocm/commands/ocmcmds/common/inputs/options"
+)
+
+var _ = Describe("Input Type", func() {
+	var env *InputTest
+
+	BeforeEach(func() {
+		env = NewInputTest(npm.TYPE)
+	})
+
+	// TODO: try out a file based NPM registry for testing.
```

Comment:
do you plan to do this within this PR or are you already done?

Reply:
Oh right, thanks for reminding me. :D